### PR TITLE
fixing pre commit test config

### PR DESCRIPTION
**Pull request by eapg**
* fixing pre commit test config
*  removing unused imports

### CODE REVIEW

File: src/tests/core/engine/processors/mongo_to_postgres_order_status_history_test.py [hunk 1]
```diff
@@ -95,7 +98,9 @@ def test_load_data_successfully(self, mocked_update_last_order_status_history):
             "632b1cbdd411e2e0c2ac80e7"
         )
 
-        self.mongo_to_postgres_etl.order_status_history_controller.get_last_order_status_histories_by_order_ids.return_value = [
+        mongo_postgresql = self.mongo_to_postgres_etl
```
Comment (reynoldmorel) [inline src/tests/core/engine/processors/mongo_to_postgres_order_status_history_test.py:101]: `etl =`

File: src/tests/core/engine/processors/mongo_to_postgres_order_status_history_test.py [hunk 2]
```diff
@@ -95,7 +98,9 @@ def test_load_data_successfully(self, mocked_update_last_order_status_history):
...
+        mongo_osh_controller = mongo_postgresql.order_status_history_controller
+        mongo_osh_controller.get_last_order_status_histories_by_order_ids.return_value = [
```
Comment (reynoldmorel) [inline src/tests/core/engine/processors/mongo_to_postgres_order_status_history_test.py:103]: `controller`

File: src/tests/core/engine/processors/mongo_to_postgres_order_status_history_test.py [hunk 3]
```diff
@@ -122,6 +127,9 @@ def after_execute(_app_processor_config, _app_context):
         self.mongo_to_postgres_etl.order_status_history_controller.add.has_called_with(
             [mock.call(order_status_history_1), mock.call(order_status_history_2)]
         )
-        self.mongo_to_postgres_etl.mongo_order_status_history_controller.update_batch_to_processed.has_called_with(
+
+        mongo_to_postgresql = self.mongo_to_postgres_etl
```
Comment (reynoldmorel) [inline src/tests/core/engine/processors/mongo_to_postgres_order_status_history_test.py:131]: Same here

**Review by reynoldmorel — APPROVED** [review]

File: src/tests/core/engine/processors/mongo_to_postgres_order_status_history_test.py (hunk 1 above)
Comment (eapg) [reply src/tests/core/engine/processors/mongo_to_postgres_order_status_history_test.py:101]: done

File: src/tests/core/engine/processors/mongo_to_postgres_order_status_history_test.py (hunk 2 above)
Comment (eapg) [reply src/tests/core/engine/processors/mongo_to_postgres_order_status_history_test.py:103]: done.

File: src/tests/core/engine/processors/mongo_to_postgres_order_status_history_test.py (hunk 3 above)
Comment (eapg) [reply src/tests/core/engine/processors/mongo_to_postgres_order_status_history_test.py:131]: done.

**Pull request merged**